lib/datasource/jenkins/get.ts

```diff
@@ -0,0 +1,102 @@
+import { logger } from '../../logger';
+import { ExternalHostError } from '../../types/errors/external-host-error';
+import { clone } from '../../util/clone';
+import { Http } from '../../util/http';
+import { Release, ReleaseResult } from '../common';
+import { id } from './common';
+
+const http = new Http(id);
+const cacheTimeMin = 60;
+
+const packageInfoUrl =
+  'https://updates.jenkins.io/current/update-center.actual.json';
+
+let lastSync = new Date('2000-01-01');
+let packageReleases: Record<string, ReleaseResult> = Object.create(null);
+
+// Note: use only for tests
+export function resetCache(): void {
+  lastSync = new Date('2000-01-01');
+  packageReleases = Object.create(null);
+}
+
+function processPlugin(name: string, data: Record<string, any>): void {
```

Comment:
```suggestion
interface JenkinsPlugin {
  name: string;
  previousVersion?: string;
  scm?: string;
  version: string;
}

function processPlugin(name: string, data: JenkinsPlugin): void {
```

maybe more properties are required for `JenkinsPlugin`